docs/manifest.json

```diff
@@ -270,7 +270,7 @@
 		"parent": "designers-developers"
 	},
 	{
-		"title": "Contributors",
+		"title": "Contributors Guide",
```

Comment:
Should not be here, right?

Reply:
There was no other way to make build pass, if you would open PR with only docs changes introduced it would fail on tests 🤷‍♂️

Reply:
What was the problem here? Did the manifest change at some point but not get committed?

Reply:
(And sorry I'm just seeing this)

Reply:
I think I changed the title on the contributors page but did not generate the manifest and commit. We missed it because Travis was throwing false errors, we just assumed it was wrong.

I think ideally we let whatever system that needs the generated file to generate at time of need and not have to try to remember generating and committing.

Reply:
@mkaz do you commit from the command line? I think Husky should have run the check automatically too. If it didn’t that could be another thing to look into.

Reply:
yep, commit from the command line and didn't see errors there